scripts/update_docs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Nit nit. LoL.

Reply:
NixOS doesn't have `/bin/bash` lol, `/usr/bin/env bash` seems to be the safer one anyway in cases of specific environment setup has been done